private_gpt/ui/ui.py

```diff
@@ -114,25 +120,22 @@ def build_history() -> list[ChatMessage]:
 
         new_message = ChatMessage(content=message, role=MessageRole.USER)
         all_messages = [*build_history(), new_message]
+        # Add a system message to force the behaviour of the LLM
```

Comment:
Given the system prompt can be about anything, this specific comment does not apply anymore

Reply:
Updated comment to `# If a system prompt is set, add it as a system message` and added code to not add the system message if no system prompt is defined.